processes_notwindows.go

```diff
@@ -46,10 +48,9 @@ func processesFromProc() ([]ProcStat, error) {
 	for _, filename := range filenames {
 		data, err := readProcFile(filename)
 		if err != nil {
```

Comment:
Is it ok to ignore this error case completely? Only if err is `errorProcessTerminated` we log it.

Reply:
we log if it NOT errorProcessTerminated. 
errorProcessTerminated means that process was terminated after we got the directory listing
This is totally OK